### DISCUSSION TITLE
Cookies

### DISCUSSION THREAD

**Discussion by Notadumdum** [miguelgrinberg/python-socketio#1120] [Q&A] [answered]
For socket.io connections that use websockets instantly without starting off with polling;would cookies be considered as custom headers, if so could http request headers be considered as custom headers as well or there's an exception if I want to make use of it on the client side.. the main question is would headers not be able to be used at all?

**Comment by miguelgrinberg** [comment, marked as answer]
Cookies are not sent via custom headers, they use the `Cookie` header which is defined in the HTTP spec. I believe all browsers send cookies when they make a WebSocket connection.

**Reply by Notadumdum** [reply]
Can cookies be sent when using a nodejs socket.io client sir?

**Reply by miguelgrinberg** [reply]
That is not my project, so I can't really give you definitive answers. But I assume you can send your cookies, since they are request headers after all.
